packages/server/src/core/degree_status/postprocessing.rs

```diff
@@ -0,0 +1,71 @@
+use crate::{
+    core::messages,
+    resources::course::{Grade, Tag},
+};
+
+use super::DegreeStatus;
+
+pub const TECHNICAL_ENGLISH_ADVANCED_B: &str = "324033";
+const EXEMPT_COURSES_COUNT_DEMAND: usize = 2;
+const ADVANCED_B_COURSES_COUNT_DEMAND: usize = 1;
+
+impl DegreeStatus {
+    fn check_english_requirement(&mut self) {
+        let completed_english_content_courses_count = self
+            .course_statuses
+            .iter()
+            .filter(|course_status| {
+                course_status
+                    .course
+                    .tags
+                    .clone()
+                    .unwrap_or_default()
+                    .contains(&Tag::English)
+                    && course_status.completed()
+            })
+            .count();
+
```

Comment:
Suggestion:
Since we have access to the `catalog` here we should check if the year is bigger than "תשפב" (which is 21/22), and return otherwise